src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -67,7 +72,7 @@ interface SnapshotOverviewPanelProps extends WithStyles<typeof styles> {
 function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
   const [value, setValue] = React.useState(0);
   const { classes, dispatch, snapshot, userRoles } = props;
-  const canEditDescription = userRoles.includes(SnapshotRoles.STEWARD);
+  const isSteward = userRoles.includes(SnapshotRoles.STEWARD);
```

Comment:
This is where I'm starting to make changes for attributes and display decisions.

Because the user is a Snapshot Steward, they:
canEdit the Description in the DescriptionView
can see the SnapshotWorkspace components.